src/Microsoft.AspNetCore.Mvc.Core/ApiExplorer/FindAllApiDescriptionProvider.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Mvc.ApiExplorer
+{
+    public class FindAllApiDescriptionProvider : DefaultApiDescriptionProfile
```

Comment:
I mocked up a few of these for the design PR. We have to have enough 'inclusive' profiles that you don't need to write your own commonly.

Reply:
`FindAllApiDescriptionProfile`